mantis-runtime-loader/src/main/java/io/mantisrx/runtime/loader/config/WorkerConfiguration.java

```diff
@@ -38,15 +38,15 @@ public interface WorkerConfiguration extends CoreConfiguration {
     // ------------------------------------------------------------------------
     //  Task Executor machine related configurations
     // ------------------------------------------------------------------------
-    @Config("mantis.taskexecutor.id")
+    @Config({"mantis.taskexecutor.id", "MANTIS_AGENT_ID"})
     @DefaultNull
     String getTaskExecutorId();
 
     default String getTaskExecutorHostName() {
         return getExternalAddress();
     }
 
-    @Config("mantis.taskexecutor.cluster-id")
+    @Config({"mantis.taskexecutor.cluster-id", "MANTIS_RESOURCE_CLUSTER_ID"})
```

Comment:
i think keeping consistent and use `MANTIS_TASKEXECUTOR_CLUSTER_ID` would be better